plaso/parsers/spotlight_storedb.py

```diff
@@ -0,0 +1,1050 @@
+        'elements_data_size': data_size})
+
+    try:
+      array_of_values = data_type_map.MapByteStream(
+          data[bytes_read:bytes_read + data_size], context=context)
+
+    except dtfabric_errors.MappingError as exception:
+      raise errors.ParseError(
+          'Unable to parse array of string values with error: {0!s}'.format(
+              exception))
+
+    if property_type & 0x03 == 0x03:
+      value = array_of_values[0]
+      if '\x16\x02' in value:
+        value = value.split('\x16\x02')[0]
+    elif property_type & 0x03 == 0x02:
+      value = array_of_values
+    else:
+      value = array_of_values[0]
+
+    bytes_read += data_size
+
+    return value, bytes_read
+
+  def _ReadMetadataAttributeVariableSizeIntegerValue(self, property_type, data):
+    """Reads a metadata attribute variable size integer value.
+
+    Args:
+      property_type (int): metadata attribute property type.
+      data (bytes): data.
+
+    Returns:
+      tuple[object, int]: value and number of bytes read.
+    """
+    if property_type & 0x02 == 0x00:
+      return self._ReadVariableSizeInteger(data)
+
+    data_size, bytes_read = self._ReadVariableSizeInteger(data)
+
+    array_of_values = []
+
+    data_offset = 0
+    while data_offset < data_size:
+      integer_value, integer_value_size = self._ReadVariableSizeInteger(
+          data[data_offset:data_size])
+
+      data_offset += integer_value_size
+
+      array_of_values.append(integer_value)
+
+    bytes_read += data_size
+
+    return array_of_values, bytes_read
+
+  def _ReadPropertyPage(self, file_object, file_offset, property_table):
+    """Reads a property page.
+
+    Args:
+      file_object (file): file-like object.
+      file_offset (int): file offset.
+      property_table (dict[int, object]): property table in which to store the
+          property page values.
+
+    Returns:
+      tuple[spotlight_store_db_property_page_header, int]: page header and next
+          property page block number.
+
+    Raises:
+      ParseError: if the property page cannot be read.
+    """
+    page_header, bytes_read = self._ReadPropertyPageHeader(
+        file_object, file_offset)
+
+    if page_header.property_table_type not in (
+        0x00000011, 0x00000021, 0x00000041, 0x00000081):
+      raise errors.ParseError(
+          'Unsupported property table type: 0x{0:08x}'.format(
+              page_header.property_table_type))
+
+    page_data = file_object.read(page_header.page_size - bytes_read)
+
+    data_type_map = self._GetDataTypeMap(
+        'spotlight_store_db_property_values_header')
+
+    file_offset += bytes_read
+
+    page_values_header = self._ReadStructureFromByteStream(
+        page_data, file_offset, data_type_map)
+
+    if page_header.property_table_type in (0x00000011, 0x00000021):
+      self._ReadPropertyPageValues(page_header, page_data, property_table)
+
+    elif page_header.property_table_type == 0x00000081:
```

Comment:
What about 0x00000041 ?

Reply:
At the moment I don't know, I don't have a sample with 0x00000041 or more data about it.